STYLE_GUIDE.md

```diff
@@ -109,6 +109,39 @@ The project's slide source code is hosted in `open-machine-learning-jupyter-book
 
 * [How to give attribution? | Creative Commons](https://creativecommons.org/use-remix/attribution/)
 
+## HTML iframe
```

Comment:
I think this is more like a guide about how to add a reusable HTML snippet in the markdown. It is not directly related to iframe. Or I will say 'HTML in Markdown through iframe'